common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -504,23 +526,40 @@ def get_policy(usage_id):
             if isinstance(course_descriptor, ErrorDescriptor):
                 return course_descriptor
 
-            # NOTE: The descriptors end up loading somewhat bottom up, which
-            # breaks metadata inheritance via get_children().  Instead
-            # (actually, in addition to, for now), we do a final inheritance pass
-            # after we have the course descriptor.
-            compute_inherited_metadata(course_descriptor)
-
-            # now import all pieces of course_info which is expected to be stored
-            # in <content_dir>/info or <content_dir>/info/<url_name>
-            self.load_extra_content(system, course_descriptor, 'course_info', self.data_dir / course_dir / 'info', course_dir, url_name)
+            if not library:
+                # NOTE: The descriptors end up loading somewhat bottom up, which
+                # breaks metadata inheritance via get_children().  Instead
+                # (actually, in addition to, for now), we do a final inheritance pass
+                # after we have the course descriptor.
+                compute_inherited_metadata(course_descriptor)
```

Comment:
That said, I'm not sure if that's a viable path forward for this PR.

Can you set up a test of inheritance in Libraries, so that we at least have a record of the behavior?